### PR TITLE
Scan: Revamp connect modal

### LINKED ISSUE #2710: Revamp connect modal

### PR DESCRIPTION
**Pull request by Pitchanai**
Fixed: #2710 

## Implementation details
- Revamp connect modal

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

**Comment by pzshine** [comment]
image

Could you re-check the font-size that whether matches the mockup or not?

File: scan/src/components/modal/connect/ConnectModal.re [hunk 1]
```diff
@@ -6,52 +6,34 @@ module Styles = {
...
+  let warning = style([display(`flex), flexDirection(`row)]);
 
   let header = active =>
     style([
       display(`flex),
       flexDirection(`row),
       alignSelf(`center),
       alignItems(`center),
-      justifyContent(`spaceBetween),
       width(`px(480)),
```
Comment (pzshine) [inline scan/src/components/modal/connect/ConnectModal.re]: I think we can remove this `width`

File: scan/src/components/modal/connect/ConnectModal.re [hunk 2]
```diff
@@ -124,55 +98,62 @@ module LoginMethod = {
 let make = (~chainID) => {
   let (loginMethod, setLoginMethod) = React.useState(_ => Mnemonic);
   <div className=Styles.container>
-    <div className=Styles.bg />
     <div className=Styles.innerContainer>
-      <VSpacing size=Spacing.xxl />
+      <VSpacing size=Spacing.xl />
```
Comment (pzshine) [inline scan/src/components/modal/connect/ConnectModal.re]: We can add `paddingTop` on Styles.innerContainer instead

File: scan/src/components/modal/connect/ConnectWithLedger.re [hunk 3]
```diff
@@ -6,7 +6,7 @@ module Styles = {
       display(`flex),
       flexDirection(`column),
       width(`percent(100.)),
-      padding4(~top=`px(18), ~left=`px(18), ~right=`px(28), ~bottom=`px(0)),
+      padding4(~top=`px(0), ~left=`px(18), ~right=`px(18), ~bottom=`px(0)),
```
Comment (pzshine) [inline scan/src/components/modal/connect/ConnectWithLedger.re]: Can use `padding2` instead

File: scan/src/components/modal/connect/ConnectWithMnemonic.re [hunk 4]
```diff
@@ -6,24 +6,16 @@ module Styles = {
       display(`flex),
       flexDirection(`column),
       width(`percent(100.)),
-      padding4(~top=`px(45), ~left=`px(18), ~right=`px(20), ~bottom=`px(0)),
+      padding4(~top=`px(0), ~left=`px(18), ~right=`px(18), ~bottom=`px(0)),
```
Comment (pzshine) [inline scan/src/components/modal/connect/ConnectWithMnemonic.re]: use `padding2` instead

**Review by pzshine — CHANGES_REQUESTED** [review]

File: scan/src/components/modal/connect/ConnectWithLedger.re [hunk 5]
```diff
@@ -142,7 +122,7 @@ let make = (~chainID, ~ledgerApp) => {
   let (result, setResult) = React.useState(_ => Nothing);
   let (accountIndex, setAccountIndex) = React.useState(_ => 0);
 
-  let createLedger = accountIndex => {
+  let createLedger = aCcccountIndex => {
```
Comment (evilpeach) [inline scan/src/components/modal/connect/ConnectWithLedger.re]: typo

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by pzshine — APPROVED** [review]

File: scan/src/components/EnhanceTxInput.re [hunk 6]
```diff
@@ -138,8 +126,27 @@ module Loading = {
...
   let make = (~msg, ~width) => {
     <div className=Styles.container>
-      <Text value=msg size=Text.Lg spacing={Text.Em(0.03)} nowrap=true block=true />
-      <LoadingCensorBar width height=30 isRight=true />
+      <Heading value=msg size=Heading.H5 marginBottom=8 align=Heading.Left />
+      <LoadingCensorBar width height=37 />
+    </div>;
+  };
+};
+
```
Comment (evilpeach) [inline scan/src/components/EnhanceTxInput.re]: Can we replace `Loading` with `Loading2` ?

File: scan/src/components/modal/connect/ConnectModal.re [hunk 7]
```diff
@@ -124,55 +98,61 @@ module LoginMethod = {
 let make = (~chainID) => {
   let (loginMethod, setLoginMethod) = React.useState(_ => Mnemonic);
   <div className=Styles.container>
-    <div className=Styles.bg />
     <div className=Styles.innerContainer>
-      <VSpacing size=Spacing.xxl />
       <div className=Styles.modalTitle>
-        <Text value="Connect With Your Wallet" weight=Text.Bold size=Text.Xxxl />
+        <Text value="Connect with your wallet" weight=Text.Medium size=Text.Xl />
         {chainID == "band-wenchang-mainnet"
```
Comment (evilpeach) [inline scan/src/components/modal/connect/ConnectModal.re]: change it to `band-guanyu-mainnet`

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/components/modal/connect/ConnectWithMnemonic.re [hunk 8]
```diff
@@ -6,24 +6,16 @@ module Styles = {
       display(`flex),
       flexDirection(`column),
       width(`percent(100.)),
-      padding4(~top=`px(45), ~left=`px(18), ~right=`px(20), ~bottom=`px(0)),
+      padding2(~v=`px(0), ~h=`px(18)),
```
Comment (evilpeach) [inline scan/src/components/modal/connect/ConnectWithMnemonic.re]: replace px(0) with zero

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Pull request merged**